django_project/base/views/project.py

```diff
@@ -289,3 +342,253 @@ def get_redirect_url(self, slug):
         project.approved = True
         project.save()
         return reverse(self.pattern_name)
+
+
+@login_required
+@csrf_exempt
+def paymentview(request):
+    '''
+    :param request:
+    :return:
+    To do
+    1. Try catch blocks
+    2. Link to javascript form
+    3. Add webhooks
+    '''
+    loggeduser = request.user.username
+    stripe.api_key = settings.STRIPE_KEY
+    ourclientid = settings.OUR_CLIENT_ID
+    myaccountid = settings.MY_ACCOUNT_ID
```

Comment:
Can you please take a look at https://github.com/inasafe/inasafe/wiki/Coding-Standards and apply these to your code - in particular variable naming should be python underscore_separated ...